test/component/uk/gov/hmrc/individualsemploymentsapi/VersioningSpec.scala

```diff
@@ -81,34 +83,11 @@ class VersioningSpec extends BaseSpec {
         """)
     }
 
-    scenario("Requests with an accept header version 2.0") {
```

Comment:
One point to note is that where we are removing sandbox tests are there sufficient tests to cover the remaining endpoint? I.e. we need to be careful we do not drop coverage where the sandbox ix used to cover shared logic.

Reply:
We have integration tests that cover the live controller. My main problem here was this was meant for versioning, hence i modified the test the check the versioning logic based on the request which still validates it will go to V2 given the corect headers.

Reply:
👍